fixtures/entrypoints-rpc-tests/tests/entrypoints.spec.ts

```diff
@@ -529,6 +545,15 @@ test("should support binding to Durable Object in another worker", async ({
 		// Check protocol, host, and cf preserved
 		expect(text).toBe('POST https://placeholder:9999/ {"thing":true}');
 	});
+
+	const rpcResponse = await fetch(new URL("/rpc", url));
+	const errors = await rpcResponse.json();
+	expect(errors).toMatchInlineSnapshot(`
+		[
+		  "Error: Cannot access \`ThingObject#property\` as Durable Object RPC is not yet supported between multiple \`wrangler dev\` sessions. We recommend you only bind to Durable Objects defined in the same Worker. You can define an entrypoint sub-classing \`WorkerEntrypoint\` to expose an RPC interface between Workers.",
+		  "Error: Cannot access \`ThingObject#method\` as Durable Object RPC is not yet supported between multiple \`wrangler dev\` sessions. We recommend you only bind to Durable Objects defined in the same Worker. You can define an entrypoint sub-classing \`WorkerEntrypoint\` to expose an RPC interface between Workers.",
```

Comment:
I find these other points either unnecessary.

1. Asking them to move the DO to a different Worker is probably untenable — that would be the default setup, so they're only doing cross-Worker because they have to (e.g. Pages -> DOs).
2. Offering `WorkerEntrypoint`s as an alternative to DOs is only giving them RPC. I think it's very unlikely that the thing they _need_ is RPC — rather, the thing they need is probably DO state or alarms or whatever. RPC is just a nice interface layer that happens to be available in both entrypoints and DOs.

```suggestion
		  "Error: Cannot access \`ThingObject#property\` as Durable Object RPC is not yet supported between multiple \`wrangler dev\` sessions.",
		  "Error: Cannot access \`ThingObject#method\` as Durable Object RPC is not yet supported between multiple \`wrangler dev\` sessions.",
```

Reply:
Oh, gotcha. That wasn't clear to me on reading the code, but perhaps with the context of a proper project in front of me, I would have understood.

Binding to a different Worker's service, rather than directly to a Durable Object namespace owned by a different Worker is, I'd say, "soft recommended". We don't actually say that anywhere, to my knowledge, and we certainly don't warn about it anywhere. So it definitely is possible to, in a first-class way, bind to those other Durable Object namespaces (without it, Pages couldn't use Durable Objects at all).

Is it likely we'll eventually be able to support Durable Object RPC across multiple sessions? I'd imagine, yes? In which case, I'd still personally suggest trimming off those final two sentences from this error message for now.

Reply:
Can we add a hidden entrypoint to `bound` which proxies to `bound`'s Durable Object? The error message says they can do it themselves but, if binding directly to DOs from another worker works in production, shouldn't we hook this up for them?

Reply:
Okay, I'm just going to commit my suggestion here then as-is, and we can see what user feedback looks like :)